extension/src/popup/components/sendPayment/SendAmount/index.tsx

```diff
@@ -257,127 +271,155 @@ export const SendAmount = ({ previous }: { previous: ROUTES }) => {
+                    formik.setFieldValue("amount", formatAmount(e.target.value))
                   }
+                  autoFocus
+                  autoComplete="off"
                 />
-              </>
-            )}
-          </div>
-
-          <div className="SendPayment__btn-continue">
-            <Button
-              disabled={
-                loadingRate || formik.values.amount === "0" || !formik.isValid
-              }
-              fullWidth
-              variant={Button.variant.tertiary}
-              type="submit"
-            >
-              Continue
-            </Button>
-          </div>
-        </form>
+                <div className="SendAmount__input-amount__asset-copy">
+                  {getAssetFromCanonical(formik.values.asset).code}
+                </div>
+                {showSourceAndDestAsset && formik.values.amount !== "0" && (
+                  <ConversionRate
+                    loading={loadingRate}
+                    source={getAssetFromCanonical(formik.values.asset).code}
+                    sourceAmount={formik.values.amount || defaultSourceAmount}
+                    dest={
+                      getAssetFromCanonical(formik.values.destinationAsset).code
+                    }
+                    destAmount={destinationAmount}
+                  />
+                )}
+                <div
+                  className={`SendAmount__amount-warning${
+                    destinationAsset ? "__path-payment" : ""
+                  }`}
+                >
+                  <DecideWarning />
+                </div>
+                <div className="SendAmount__asset-select-container">
+                  {!showSourceAndDestAsset && (
+                    <AssetSelect
+                      assetCode={
+                        getAssetFromCanonical(formik.values.asset).code
+                      }
+                      issuerKey={
+                        getAssetFromCanonical(formik.values.asset).issuer
+                      }
+                    ></AssetSelect>
+                  )}
+                  {showSourceAndDestAsset && (
+                    <>
+                      <PathPayAssetSelect
+                        source={true}
+                        assetCode={
+                          getAssetFromCanonical(formik.values.asset).code
+                        }
+                        issuerKey={
+                          getAssetFromCanonical(formik.values.asset).issuer
+                        }
+                        balance={formik.values.amount}
+                      />
+                      <PathPayAssetSelect
+                        source={false}
+                        assetCode={
+                          getAssetFromCanonical(formik.values.destinationAsset)
+                            .code
+                        }
+                        issuerKey={
+                          getAssetFromCanonical(formik.values.destinationAsset)
```

Comment:
might be useful to create const's for ` getAssetFromCanonical(formik.values.destinationAsset)` and `getAssetFromCanonical(formik.values.asset)` as we're calling these 2 methods a few time